app/commands/mentor/update_stats.rb

```diff
@@ -3,9 +3,7 @@ class Mentor::UpdateStats
 
   queue_as :default
 
-  initialize_with :mentor,
-    update_counts: false,
-    update_satisfaction_rating: false
```

Comment:
We can't use these fields in the new way, but we don't actually need to, as the new code (using `sum` is more efficient, so we don't have the pain we used to).